web/themes/joinup/prototype/sass/components/_search-bar.sass

```diff
@@ -70,7 +70,6 @@
     width: 200px
   .search-bar__input
     font-weight: $font-weight-thin
-    padding: 7px 0
```

Comment:
Looking at this selector for this change, I see further refactoring could be possible:

![search-input](https://user-images.githubusercontent.com/1923476/28420373-363c6204-6d62-11e7-9a2d-43167240cc0c.png)